PopPUNK/sparse_mst.py

```diff
@@ -127,7 +128,20 @@ def main():
             for v in mst.vertices:
                 isolateClustering['Cluster'][mst.vp.id[v]] = '0'
 
-        drawMST(mst, args.output, isolateClustering, True)
+        # Check selecting clustering type is in CSV
+        clustering_name = 'Cluster'
+        if args.display_cluster != None and args.previous_clustering != None:
+            if args.display_cluster not in isolateClustering.keys():
+                sys.stderr.write('Unable to find clustering column ' + args.display_cluster + ' in file ' +
+                                 args.previous_clustering + '\n')
+                sys.exit()
```

Comment:
Better to give null clusters than exit I think, as there might have been quite a lot of processing up to this point

Reply:
Changed in 6aba143.